parser.c

```diff
@@ -149,6 +166,10 @@ static bool parser_next_token(struct kanshi_parser *parser) {
 			parser->tok_type = KANSHI_TOKEN_STR;
 			parser->tok_str_len = 0;
 			return parser_read_quoted(parser);
+		} else if (ch == commentSign) {
+			parser->tok_type = KANSHI_TOKEN_COMMENT;
+			parser->tok_str_len = 0;
+			return true;
```

Comment:
What I mean is that you can `parser_ignore_line` instead of returning here. This would save us from having to have `KANSHI_TOKEN_COMMENT`.

Reply:
Changing this disable support for
`
output ...# comment at the end of line
`

Reply:
Works with this patch: https://sr.ht/3tRr.txt

We can get rid of the comment token.